### PR TITLE
feat: Add exchange per neighbor ga view

### PR DESCRIPTION
**Pull request by johevemi**
# Description

https://energinet.atlassian.net/wiki/spaces/D3/pages/854491137/eSett+Result+Model#Exchange-Per-Neighbor-Grid-Area

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__calculation_results__energy_result_per_neighbor_ga_v1.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+CREATE VIEW {CALCULATION_RESULTS_DATABASE_NAME}.exchange_per_neighbor_ga_v1 as
+SELECT calculation_id,
+       calculation_type,
+       grid_area_code,
+       calculation_period_start,
+       calculation_period_end,
+       calculation_version,
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__calculation_results__energy_result_per_neighbor_ga_v1.sql:7]: nit: I think it makes sense to have all the calculation columns first
```suggestion
       calculation_period_start,
       calculation_period_end,
       calculation_version,
       grid_area_code,
```
The comment applies to all

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__calculation_results__energy_result_per_neighbor_ga_v1.sql
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__calculation_results__energy_result_per_neighbor_ga_v1.sql:1]: Should grid_area_code be replaced by out_grid_area_code and in_grid_area_code?

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__calculation_results__energy_result_per_neighbor_ga_v1.sql (hunk 1 above)
Comment (johevemi) [reply source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__calculation_results__energy_result_per_neighbor_ga_v1.sql:7]: okay, i will update confluence as well

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__calculation_results__energy_result_per_neighbor_ga_v1.sql
Comment (johevemi) [reply source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__calculation_results__energy_result_per_neighbor_ga_v1.sql:1]: yes

**Review by BjarkeMeier — APPROVED** [review]

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**